src/java/org/apache/cassandra/db/MultiClusteringBuilder.java

```diff
@@ -403,6 +403,9 @@ public NavigableSet<Clustering<?>> build()
     {
         built = true;
 
+        if (clusterings.size() == 1 && clusterings.get(0) == ClusteringElements.ROOT)
+            return BTreeSet.of(comparator, Clustering.EMPTY);
+
```

Comment:
Shouldn't we add the same thing in `buildBound` for consistency?

Reply:
Indeed, good catch. I have added a similar check returning an empty set of bounds.